src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * 40;
+
+  if (days < 3) {
+    return price;
+  } else if (days <= 6) {
```

Comment:
4. [Checklist] Prefer if with return over if else to simplify later conditions.

Reply:
done